x-pack/plugins/ml/public/settings/filter_lists/list/directive.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import 'ngreact';
+
+import { uiModules } from 'ui/modules';
+const module = uiModules.get('apps/ml', ['react']);
+
+import { checkLicense } from 'plugins/ml/license/check_license';
+import { checkGetJobsPrivilege } from 'plugins/ml/privilege/check_privilege';
+import { getMlNodeCount } from 'plugins/ml/ml_nodes_check/check_ml_nodes';
+import { initPromise } from 'plugins/ml/util/promise';
+
+import uiRoutes from 'ui/routes';
+
+const template = `
+  <ml-nav-menu name="settings"></ml-nav-menu>
+  <div class="ml-filter-lists">
+    <ml-filter-lists></ml-filter-lists>
+  </div>
+`;
+
+uiRoutes
+  .when('/settings/filter_lists', {
+    template,
+    resolve: {
+      CheckLicense: checkLicense,
+      privileges: checkGetJobsPrivilege,
+      mlNodeCount: getMlNodeCount,
+      initPromise: initPromise(true)
+    }
+  });
+
+
+import { FilterLists } from './filter_lists';
+
+module.directive('mlFilterLists', function ($injector) {
+  const reactDirective = $injector.get('reactDirective');
```

Comment:
the edit page uses `ReactDOM` and this page uses `reactDirective`.
we should probably use one or the other.
I'm leaning more towards `ReactDOM` these days :)